src/Api/Endpoints/Signin.cs

```diff
@@ -8,18 +8,16 @@ namespace Passwordless.Api.Endpoints;
 
 public static class SigninEndpoints
 {
-    private record SigninTokenRequest(string UserId);
-
-    private record SigninTokenResponse(string Token);
+    public record SigninTokenResponse(string Token);
 
     public static void MapSigninEndpoints(this WebApplication app)
     {
         app.MapPost("/signin/generate-token", async (
-                SigninTokenRequest signinToken,
+                SignInTokenRequest signinToken,
```

Comment:
Nit

Seems like we're slightly inconsistent on the casing of `Signin`/`SignIn`. I thought, when used as a noun, `Signin` is more common. But either way, we should probably stick to one convention.